core/src/subgraph/registrar.rs

```diff
@@ -939,3 +949,44 @@ fn reassign_subgraph(
 
     Ok(())
 }
+
+fn update_subgraph_query_prices(
```

Comment:
See my comments in #1198. I think query prices don't belong into deployments directly, but rather a separate `SubgraphDeploymentPricing` entity or similar.